t/unit/utils/test_time.py

```diff
@@ -278,7 +271,7 @@ def test_localize_aware_dt_idempotent(self):
         assert localized_utc_offset == local_zone.utcoffset(local_time)
 
 
-@pytest.mark.parametrize('s,expected', [
+@ pytest.mark.parametrize('s,expected', [
```

Comment:
There are a few spaces that sneaked into this file :)
Check below as well

Reply:
Fixed!